adapters/lemmadigital/lemmadigital.go

```diff
@@ -102,7 +102,7 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 			}
 			bidResponse.Bids = append(bidResponse.Bids, b)
 		}
-		break
+		break //nolint: staticcheck // break is intended, exempt from staticcheck SA4004
```

Comment:
Instead of the `nolint` tag, can we remove the `break` entirely?
```
 97 -     for _, seatBid := range response.SeatBid {
 98 -         for i := range seatBid.Bid {
    +     if len(response.SeatBid) > 0 {
    +         for i := range response.SeatBid[0].Bid {
 99               b := &adapters.TypedBid{
100 -                 Bid:     &seatBid.Bid[i],
    +                 Bid:     &response.SeatBid[0].Bid[i],
101                   BidType: bidType,
102               }
103               bidResponse.Bids = append(bidResponse.Bids, b)
104           }
105 -         break //nolint: staticcheck // break is intended, exempt from staticcheck SA400
106       }
107  
108       return bidResponse, nil
109   }
adapters/lemmadigital/lemmadigital.go [RO]
```

Reply:
done in 6d4f03aa

Reply:
Thank you